src/Model/MenuSet.php

```diff
@@ -6,14 +6,20 @@
 use SilverStripe\ORM\DataObject;
 use SilverStripe\Forms\TextField;
 use SilverStripe\Security\Permission;
-use SilverStripe\Forms\GridField\GridField;
-use SilverStripe\Forms\GridField\GridFieldConfig_RecordEditor;
+use SilverStripe\LinkField\Form\MultiLinkField;
+use SilverStripe\LinkField\Models\ExternalLink;
+use SilverStripe\LinkField\Models\Link;
+use SilverStripe\LinkField\Models\SiteTreeLink;
 use SilverStripe\Security\PermissionProvider;
 use SilverStripe\Subsites\Model\Subsite;
-use Symbiote\GridFieldExtensions\GridFieldOrderableRows;
+use SilverStripe\Versioned\Versioned;
 
 class MenuSet extends DataObject implements PermissionProvider
 {
+    private static $extensions = [
```

Comment:
[nitpick] since this is a new property

```suggestion
    private static array $extensions = [
```